src/aggregation/metric/mod.rs

```diff
@@ -92,8 +96,9 @@ pub struct TopHitsVecEntry {
 
     /// Search results, for queries that include field retrieval requests
     /// (`docvalue_fields`).
-    #[serde(flatten)]
-    pub search_results: FieldRetrivalResult,
```

Comment:
this should have been `FieldRetrievalResult` too. 

Is it still used? Can we remove it?

Reply:
It can't be `FieldRetrievalResult`, since postcard does not support flatten